pkg/azuredisk/azuredisk.go

```diff
@@ -26,20 +26,19 @@ import (
 	"time"
 	"unicode"
 
+	csicommon "sigs.k8s.io/azuredisk-csi-driver/pkg/csi-common"
```

Comment:
hi @chewong, are you going to do the same repo address rename for azurefile-csi-driver? We may make it consistent for these two drivers, thanks.

Reply:
Opened https://github.com/kubernetes-sigs/azurefile-csi-driver/pull/127 to address this issue.